testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
I just realized reading the spec at http://haxe.org/manual/class-field-property.html ... that, 

get/set: Access is generated as a call to an accessor method. The compiler ensures that the accessor is available.

i.e. no need to prefix it with either public or private !
Same is confirmed by examples in http://haxe.org/manual/class-field-property-common-combinations.html and http://haxe.org/manual/class-field-property-rules.html